### ISSUE TITLE
Improve the workflomics visualisation of the metrics 

### ISSUE THREAD

**Issue by vedran-kasalica** [Workflomics/pubmetric#48]
the current visualization presents the metrics as 2 independent values:
![Screenshot 2024-10-25 at 17 51 10](https://github.com/user-attachments/assets/0fa999c3-d4d2-4f07-9d01-ee6f28a78ce3)

Proposed improvements:

- [ ] Group the 2 values together
- [ ] Rename `Pubmetric` to a more descriptive metric name (e.g., co-citations)

**Comment by magnuspalmblad** [comment]
The OS compatibility should also be grouped together I think. Maybe just with a thin horizontal line between categories?

**Comment by vedran-kasalica** [comment]
@magnuspalmblad I agree. I added the issue under Workflomics front-end (see https://github.com/Workflomics/workflomics-frontend/issues/80) as it was not pubmetric-specific.

**Comment by magnuspalmblad** [comment]
Can this issue be closed now?

**Comment by vedran-kasalica** [comment]
@magnuspalmblad thanks for pointing it out! The issue was resolved
